packages/terra-date-picker/src/DateUtil.jsx

```diff
@@ -0,0 +1,33 @@
+import moment from 'moment';
```

Comment:
Updated the file from .jsx to .js. https://github.com/cerner/terra-core/pull/290/commits/ea493813ebb2ba2c613dd5152cf3fa7d224a11f3